ChangeLog

```diff
@@ -1,3 +1,10 @@
+2025-04-08  Dirk Eddelbuettel  <edd@debian.org>
```

Comment:
Well this is wrong because the commit is yours not mine.

Can we simplify and make it more truthful, ie you add a Changelog entry about the changes to inst/include/* and if possibly make no other changes?

A patch for sprintf -> snprint is sensible too, it may make sense to keep it in a local file (say `local/change_to_1.28.2.patch` or something like that,

Reply:
I've limited this now to the library update and the patch.

For the patch, I think having the mention in the Changelog should be enough. It doesn't look like we update this library very often and down the line it may not even be needed - as long we run revdepchecks, we'd catch any future omission.

I'm not seeing a great benefit to keeping it in another file as we'd still need to patch `asio.hpp` to include it I suppose.

Reply:
I still like to see the patch simply as a reminder.  

And I don't mean in another file to be included.  Simply as a 'note' of changes made due to CRAN.  Look at my BH repo, I keep them there too (also to reapply them, of course, but it is more "industrial sized" there).

Reply:
Right, got it. Done in [a22fb6c](https://github.com/eddelbuettel/asioheaders/pull/11/commits/a22fb6c39863bf8e8e2418e6b9439f9c725476d9).